workos/sso.py

```diff
@@ -86,6 +87,25 @@ def get_authorization_url(
 
         return prepared_request.url
 
+    def get_profile(self, accessToken):
+        """
+        Verify that SSO has been completed successfully and retrieve the identity of the user.
+
+        Args:
+            accessToken (str): the token used to authenticate the API call
+
+        Returns:
+            WorkOSProfile
+        """
+
+        token = {"access_token": accessToken}
+
+        response = self.request_helper.request(
+            PROFILE_PATH, method=REQUEST_METHOD_GET, token=token
```

Comment:
I think this should just be `token=accessToken` rather than passing in a dictionary.

Reply:
Fixed, thanks!